rtos/rtx/TARGET_CORTEX_M/cmsis_os.h

```diff
@@ -69,11 +69,15 @@
 #endif
 
 // The stack space occupied is mainly dependent on the underling C standard library
+#ifdef OS_MAINSTKSIZE
+#define WORDS_STACK_SIZE OS_MAINSTKSIZE
+#else
```

Comment:
Is this change needed for this pr? I believe there is quite a bit of discussion around the main stack size already.